blocks/nav/__item/nav__item.css

```diff
@@ -0,0 +1,18 @@
+.nav__item {
```

Comment:
According to the layout, these links have ':hover' effect, and in this position it is different then in desktop dimension. Check it, please. It'd be nice to lead the pixel perfect concept